gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -221,8 +221,9 @@ public GsonBuilder serializeNulls() {
    * on the key; however, when this is called then one of the following cases
    * apply:
    *
-   * <h3>Maps as JSON objects</h3>
-   * For this case, assume that a type adapter is registered to serialize and
+   * <p><b>Maps as JSON objects</b>
```

Comment:
The proper heading would be `<h4>`, but probably due to [JDK-8220379](https://bugs.openjdk.org/browse/JDK-8220379) `javadoc` of JDK 11 does not permit using it. I guess using `<b>` for now is fine though.

Reply:
Yes, `<h3>` was rejected on JDK 11 and `<h4>` on JDK 17 (or maybe the other way around), so I did this annoying thing instead.